js/currencycom.js

```diff
@@ -987,6 +1020,77 @@ module.exports = class currencycom extends Exchange {
         }, market);
     }
 
+    parseOrderStatus (status, inverted = false) {
+        let statuses = {
+            'NEW': 'open',
+            'PARTIALLY_FILLED': 'open',
+            'FILLED': 'closed',
+            'CANCELED': 'canceled',
+            'PENDING_CANCEL': 'canceling',
+            'REJECTED': 'rejected',
+            'EXPIRED': 'expired',
+        };
+        if (inverted) {
+            statuses = {
+                'open': 'NEW',
+                'closed': 'FILLED',
+                'canceled': 'CANCELED',
+                'canceling': 'PENDING_CANCEL',
+                'rejected': 'REJECTED',
+                'expired': 'EXPIRED',
+            };
+        }
+        return this.safeString (statuses, status, status);
+    }
+
+    parseOrderType (status, inverted = false) {
+        let statuses = {
+            'MARKET': 'market',
+            'LIMIT': 'limit', // TO_DO: what is LIMIT_MAKER ?
+            'STOP': 'stop',
+            'STOP_LOSS': 'stop-loss',
```

Comment:
We don't have a unified `stop-loss` type yet. This is yet to be unified. Which is why we should propagate the non-standard order types "as is" for now.